### PR TITLE
Remove automatic :running_in_container setting

### PR DESCRIPTION
**Pull request by jeffkreeftmeijer**
Depends on https://github.com/appsignal/appsignal-agent/pull/217.

### CODE REVIEW

File: lib/appsignal/cli/diagnose.rb [hunk 1]
```diff
@@ -79,9 +79,8 @@ def host_information
           puts "  Heroku: true" if Appsignal::System.heroku?
           print "  root user: "
           puts Process.uid == 0 ? "yes (not recommended)" : "no"
-
-          return unless Appsignal::System.container?
-          puts "  Container id: #{Appsignal::System::Container.id}"
+	  print "  Running in container: "
+	  puts Appsignal::Extension.running_in_container == 1 ? "yes" : "no"
```
Comment (tombruijn) [inline lib/appsignal/cli/diagnose.rb]: indenting 🐟

File: ext/appsignal_extension.c [hunk 2]
```diff
@@ -635,6 +639,7 @@ void Init_appsignal_extension(void) {
 
   // Event hook installation
   rb_define_singleton_method(Extension, "install_allocation_event_hook", install_allocation_event_hook, 0);
+  rb_define_singleton_method(Extension, "running_in_container", running_in_container, 0);
```
Comment (tombruijn) [inline ext/appsignal_extension.c]: We can't return a boolean in a `running_in_container*?*` method?

Comment (thijsc) [inline ext/appsignal_extension.c]: You can add a wrapper method in `lib/extension.rb`:

```ruby
def running_in_container?
  running_in_container == 1
end
```

Comment (thijsc) [inline ext/appsignal_extension.c]: Ignore that, you can just call it `running_in_container?` here in the second argument. It already returns a boolean.

File: lib/appsignal/cli/diagnose.rb (hunk 1 above)
Comment (thijsc) [inline lib/appsignal/cli/diagnose.rb]: Since in returns a boolean already you don't need the `== 1`.

File: ext/appsignal_extension.c (hunk 2 above)
Comment (jeffkreeftmeijer) [inline ext/appsignal_extension.c]: Hm, I was convinced I already tried it with the question mark. 🤔 Anyway, fixed.

**Comment by jeffkreeftmeijer** [comment]
Changed base to develop, rebased on develop and removed the agent update commit, as the current agent version in develop is newer. Tested locally with the diagnose task, this one is ready for re-review.

**Pull request merged**